docs/astro/src/content/docs/reference/elements/text.mdx

```diff
@@ -258,3 +258,7 @@ Text {
 
 ### rotation-origin-y
 <SlintProperty propName="rotation-origin-y" typeName="length"/>
+
+## Accessibility
+
+By default, `Text` elements have the `accessible-role` property set to `text`.
```

Comment:
And I believe accessible-label is also set.

Reply:
Fair point :)